tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>Here, we want to again talk about both the estimator and sampler gradients. The opflow framework didn't have sampler gradients and this is why there is no reference explanation there, and we have to add it as part of this new tutorial.</p><p> </p><p>The next couple of cells could be modified to show this:</p><p> </p><blockquote>&nbsp;&nbsp;&nbsp;Given a parameterized quantum state [formula] with input state $|\\psi\\rangle$, parametrized Ansatz $V\\left(\\theta\\right)$, and observable $\\hat{O}\\left(\\omega\\right)=\\sum_{i}\\omega_i\\hat{O}_i$, we want to compute either its expectation gradient [formula already there] or the gradient of the sampling probability [new formula]</blockquote><p> </p><p>the formula for the gradient of sampling is:</p><p> </p><pre class="ql-syntax" spellcheck="false">$$\\frac{\\partial}{\\partial \\theta} p_j(\\theta) = \\frac{\\partial}{\\partial \\theta} |\\langle j | \\psi(\\theta)\\rangle|^2 $$
</pre><p> </p><p>We can also add here that the output format of the estimator gradient is a list of derivatives (pseudocode that can be added:)</p><pre class="ql-syntax" spellcheck="false">[d/d theta_1 &lt;E&gt;, d/d theta_2 &lt;E&gt;, ...]
</pre><p> </p><p>while the output of the sampler gradient is a list of dictionaries, where each dictionary has entries for the different&nbsp;<code>j</code>&nbsp;in the formula above:</p><pre class="ql-syntax" spellcheck="false">[{d/d theta_1 p_1: .., d/d theta_1 p_2, ..,}, {d/d theta_2 p_1: .., d/d theta_2 p_2, ..}, ..]
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Edited. </p>